services/handbooks/workspace/docs/domains/README.md

```
@@ -35,3 +35,45 @@
+├── components/           # UI コンポーネント
+│   ├── cast/            # Cast 向けコンポーネント
+│   └── guest/           # Guest 向けコンポーネント
+├── hooks/               # データフェッチ、状態管理
+├── lib/                 # マッパー、ユーティリティ
+└── types.ts             # ドメイン型定義
+```
+
+### Guidelines
+
+| ファイル | 役割 | 例 |
+|----------|------|-----|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)